service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -79,8 +81,14 @@
     @PostMapping("/about-to-start")
     public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
         CaseDetails caseDetails = callbackrequest.getCaseDetails();
+        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
 
-        caseDetails.getData().put("dateOfIssue", time.now().toLocalDate());
+        if (isEmpty(caseData.getStandardDirectionOrder().getDateOfIssue())) {
+            caseDetails.getData().put("dateOfIssue", time.now().toLocalDate());
+        } else {
+            caseDetails.getData().put("dateOfIssue",
+                parseLocalDateFromStringUsingFormat(caseData.getStandardDirectionOrder().getDateOfIssue(), DATE));
```

Comment:
let's have the put code once and then just set a variable to the date in the if statements

Reply:
👍